Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMeshPrimitiveAttributes.cs

```diff
@@ -9,21 +9,17 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
     /// Common mesh primitive attributes.
     /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/mesh.primitive.schema.json
     /// </summary>
-    /// <remarks>
-    /// Application specific semantics are not supported
-    /// </remarks>
-    [Serializable]
     public class GltfMeshPrimitiveAttributes
     {
-        public int POSITION = -1;
```

Comment:
Removing these public fields is a [breaking change](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html). I do think this is okay in this case though, since this feels largely like an internal implementation detail for the glTF loader. Interested to get other opinions though.

Reply:
I could re-think things a little in an effort to keep it backwards compatible. Perhaps I could change these to properties that access an internal dictionary. . .

Just saying it out loud doesn’t sound quite right. This change certainly could bite someone, but it feel like a reasonable change to make with an appropriate release.

Reply:
Yeah, agreed. I haven't fully dug in yet, but some of these possibly feel like classes that should have been `internal` to begin with. Not sure how useful this class would be outside the code that's loading the model

Reply:
Well, custom attributes is one use case that motivates making some of this public. I’m sure more thought could be put into exactly what the public interface should be.

Reply:
Well, custom attributes is one use case that motivates making some of this public. I’m sure more thought could be put into exactly what the public interface should be.

Reply:
Oh good call, yeah